googleanalytics/googleanalytics-connector/googleanalytics-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/googleanalytics-profileFilterLink/component.xml

```diff
@@ -0,0 +1,12 @@
+<component name="googleanalytics-profileFilterLink" type="synapse/template">
+    <subComponents>
+        <component name="listProfileFilterLink">
+            <file>listProfileFilterLink.xml</file>
+            <description>Retrieve a list of filters from GoogleAnalytics account.</description>
```

Comment:
This description in given for listFilters method. Change this according to the action.